Dockerfile

```diff
@@ -482,6 +482,20 @@ ENV PATH "$SWIFTENV_ROOT/bin:$SWIFTENV_ROOT/shims:$PATH"
 RUN swiftenv install ${NETLIFY_BUILD_SWIFT_VERSION}
 RUN swift --version
 
+################################################################################
+#
+# Homebrew
+#
+################################################################################
+USER buildbot
+RUN /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install.sh)"
```

Comment:
Any concern that we aren't pulling from a static/stable tag?

Reply:
that's a good point. let's do that as part of a general release